Sources/Components/StepIndicator/ViewHelpers/LineBuilder.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+extension Warp.StepIndicator {
+    struct LineBuilder {
+        @ViewBuilder
+        static func line(
+            for otherProgress: Warp.StepIndicatorItem.Progress,
+            ownProgress: Warp.StepIndicatorItem.Progress,
+            colorProvider: ColorProvider = Warp.Config.colorProvider,
+            orientation: Warp.StepIndicator.LayoutOrientation,
+            lineThickness: Double = 2
+        ) -> some View {
+
+            let fillColor = switch otherProgress {
+            case .incomplete:
+                colorProvider.token.backgroundDisabled
```

Comment:
Please use the `stepIndicatorColors` in `colorProvider` directly, for example this one can be `stepIndicatorTrackBackground`